spec/geostoreSpec.js

```diff
@@ -506,7 +506,7 @@ describe("geostore", function() {
 
         gs.within({
           "type": "Polygon",
-          "coordinates": [ [ [ -122, 45, 0.0 ], [ -123, 45, 0.0 ], [ -123, 46, 0.0 ], [ -122, 46, 0.0 ], [ -122, 45, 0.0 ] ] ]
+          "coordinates": [ [ [ -121, 45, 0.0 ], [ -123, 45, 0.0 ], [ -123, 46, 0.0 ], [ -121, 46, 0.0 ], [ -121, 45, 0.0 ] ] ]
```

Comment:
Although this check has nothing to do with the new `intersects` logic, it was failing due to the result polygon not contained by the search polygon.